src/main/java/it/gov/pagopa/pu/workflow/wf/debtposition/custom/fine/activity/InvokeSyncDebtPositionActivityImpl.java

```diff
@@ -0,0 +1,27 @@
+package it.gov.pagopa.pu.workflow.wf.debtposition.custom.fine.activity;
+
+import io.temporal.spring.boot.ActivityImpl;
+import it.gov.pagopa.payhub.activities.dto.debtposition.syncwfconfig.GenericWfExecutionConfig;
+import it.gov.pagopa.pu.debtposition.dto.generated.DebtPositionDTO;
+import it.gov.pagopa.pu.workflow.dto.PaymentEventRequestDTO;
+import it.gov.pagopa.pu.workflow.wf.debtposition.custom.fine.wfreductionexpiration.FineReductionOptionExpirationWFImpl;
+import it.gov.pagopa.pu.workflow.wf.debtposition.sync.SynchronizeDebtPositionWfClient;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+
+@Service
+@Slf4j
+@ActivityImpl(taskQueues = FineReductionOptionExpirationWFImpl.TASK_QUEUE_SYNCHRONIZE_DP_SYNC_LOCAL_ACTIVITY)
```

Comment:
let's use it.gov.pagopa.pu.workflow.wf.debtposition.sync.config.SynchronizeDebtPositionWfConfig#TASK_QUEUE_SYNCHRONIZE_DP_LOCAL_ACTIVITY
instead, in order to use always the same for all custom WF
```suggestion
@ActivityImpl(taskQueues = SynchronizeDebtPositionWfConfig.TASK_QUEUE_SYNCHRONIZE_DP_LOCAL_ACTIVITY)
```